gameswap-service/src/main/java/org/gameswap/application/GameswapConfiguration.java

```diff
@@ -35,6 +35,11 @@
     @JsonProperty
     private String jwtSecret;
 
+    @Valid
+    @NotNull
+    @JsonProperty
+    private int socketReadTimeoutMillis;
```

Comment:
Added this since I was getting many socket time out exceptions when signing in via google